frontend/src/features/missions/MissionForm/CancelEditModal.tsx

```diff
@@ -1,20 +1,49 @@
 import { Accent, Button, Dialog, THEME } from '@mtes-mct/monitor-ui'
+import { useMemo } from 'react'
 import styled from 'styled-components'
 
 type CancelEditModalProps = {
+  dirty: Boolean
```

Comment:
```suggestion
  isDirty: boolean
```
Avec un `is` car booléen et avec le type natif `boolean` ?